builtin/method_dict.py

```diff
@@ -27,3 +27,20 @@ def Call(self, rd):
 
         keys = [value.Str(k) for k in dictionary.keys()]  # type: List[value_t]
         return value.List(keys)
+
+
+class Values(vm._Callable):
+
+    def __init__(self):
+        # type: () -> None
+        pass
+
+    def Call(self, rd):
+        # type: (typed_args.Reader) -> value_t
+
+        dictionary = rd.PosDict()
+        rd.Done()
+
+        keys = [k for k in dictionary.values()]  # type: List[value_t]
```

Comment:
should be called `values` or `v` or `vals` :-)

Reply:
reminder :)

Oh also I think it should just be

```
vals = dictionary.values()
```

No need for list comprehension

In the keys() case we need the list comprehension, because we have to add the `value.Str()` wrapper